features/financial_assistance/other_questions_pregnancy.feature

```diff
@@ -25,24 +25,28 @@ Feature: Start a new Financial Assistance Application and answers questions on O
 
   Scenario: Pregnancy response within 60 days Yes with information filled and submitted
     Given the user answers no to being pregnant
+    And the post partum language one year is not enabled
```

Comment:
These lines can be replaced with:
```
And FAA post_partum_period_one_year feature is enabled
``` 
Check out the comment in the steps file for details.